js/main.js

```diff
@@ -54,13 +54,11 @@ function pad(num, totalDigits) {
 function getTimeTableHtml(openingRanges) {
     var html = '<table class="times">';
     if (openingRanges !== undefined) {
-        for (var index = 0, openingRangesLength = openingRanges.length; index < openingRangesLength; ++index) {
-            var openingRange = openingRanges[index];
-
+        openingRanges.forEach(function(openingRange) {
```

Comment:
Looks good, for maximum functional-style, you could rewrite this using `reduce` like this (not tested):
```javascript
html += openingRanges.reduce(function (tableRows, openingRange) {
  var tableRow = getTableRowForDay(openingRange, dayIsToday);
  return tableRows + getTableRowForDay(openingRange, dayIsToday);
}, '');
```

Reply:
Resolved in eefc320992052378b6fbbbc0bc11259bad517d75.